_search-plugins/knn/knn-vector-quantization.md

```diff
@@ -0,0 +1,103 @@
+---
```

Comment:
This documentation isnt complete if there isnt a reference to PQ.

Reply:
Agree, we must add PQ here. But, can we add it in a separate PR? Also, can you pls add about PQ as you know more info than me.

Reply:
Sure I can add later

Reply:
Thanks